rskj-core/src/main/resources/reference.conf

```diff
@@ -406,6 +406,8 @@ sync {
 
 rpc {
     callGasCap = 50000000
+    # Allow StateOverride in eth_call
+    allowStateOverride = false
```

Comment:
suggestion (to be a little bit more specific): `allowCallStateOverride`

Reply:
Done!